www/js/app.js

```diff
@@ -843,7 +918,7 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         request.responseType = "blob";
         request.onreadystatechange = function () {
             if (request.readyState === XMLHttpRequest.DONE) {
-                if ((request.status >= 200 && request.status < 300) || request.status === 0) {
+                if ((request.status >= 300 && request.status < 300) || request.status === 0) {
```

Comment:
@vasusharma7 Did @mossroy specifically ask you to change this? That seems a bit strange, as this function is only used by testing and has nothing to do with this PR... ? Maybe a mistake?

Reply:
Yes....Its this is my mistake.....I will mend it next time